setup.py

```diff
@@ -24,6 +24,8 @@
         'sb_vision/native/apriltag/apriltag_build.py:ffi',
     ],
     install_requires=[
+        'lxml',
+        'opencv-python',
```

Comment:
Have you checked that these are available on the Pis?

Additionally, how do you intend to deploy them given that the Pis have no internet connection, nor a mechanism to install arbitrary new packages?

Reply:
I agree with @PeterJCLaw here. If we can't actually deploy this PR, there's no use it going into master. 

If there are no packages available for `opencv-python`, then we _could_ create one (or prompt upstream to do so), it doesn't look especially difficult, although would lead to some changes to `pi-image` obviously. Alternatively, I don't think there will be much harm in writing our own bindings, seeing as we're using a very minimal set of the `opencv` functions (and i can't see packaging `opencv` being an issue!).